WooCommerce/Classes/POS/Card Present Payments/CardPresentPaymentCollectOrderPaymentUseCaseAdaptor.swift

```diff
@@ -89,6 +92,7 @@ final class CardPresentPaymentCollectOrderPaymentUseCaseAdaptor {
                 }
             } onCancel: {
                 // TODO: cancel any in-progress discovery, connection, or payment. #12869
+                invalidatablePaymentOrchestrator.invalidatePayment()
```

Comment:
Love it! Nice that it's called for all cases, for one particular payment collection task 💯